src/components/Facility/CentralNursingStation.tsx

```diff
@@ -216,21 +218,23 @@ export default function CentralNursingStation({ facilityId }: Props) {
           {t("no_vitals_present")}
         </div>
       ) : (
-        <div className="3xl:grid-cols-3 mt-1 grid grid-cols-1 gap-1 lg:grid-cols-2">
-          {data.map((props, i) => (
-            <div className="overflow-hidden text-clip" key={i}>
-              <HL7PatientVitalsMonitor
-                patientCurrentBedAssignmentDate={
-                  props.patientAssetBed?.patient?.last_consultation?.current_bed
-                    ?.start_date
-                }
-                key={`${props.patientAssetBed?.bed.id}-${hash}`}
-                patientAssetBed={props.patientAssetBed}
-                socketUrl={props.socketUrl || ""}
-                config={config}
-              />
-            </div>
-          ))}
+        <div className="@container">
```

Comment:
instead of adding container inside <Page..../> I created a new class as parent because previous implementation was causing this bug   https://github.com/ohcnetwork/care_fe/pull/9013#issuecomment-2473293072